lib/inferno/test_runner.rb

```diff
@@ -92,7 +91,7 @@ def run_test(test, scratch)
           requests: test_instance.requests,
           result: result,
           result_message: test_instance.result_message,
-          input_json: input_json_string,
+          input_json: inputs_as_json(test, inputs),
```

Comment:
I think this generation should still happen in line 64 to be safe.

Reply:
Moved this up